Lab3.ipynb

```diff
@@ -0,0 +1,401 @@
+      "text/plain": [
+       "<Figure size 1440x1080 with 2 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "a = np.vstack((stats.norm.rvs( scale = 1, size = 100000), stats.norm.rvs( scale = 1, size = 100000)))\n",
+    "a.shape\n",
+    "fig, ax = plt.subplots(1, 1)\n",
+    "h = ax.hist2d(a[0,:],a[1,:],bins=100, density=True);\n",
+    "ax.set_aspect('equal', 'box')\n",
+    "plt.xlim([-3 , 3])\n",
+    "plt.ylim([-3 , 3])\n",
+    "plt.title(\"2D Histogram of positional uncertainty\", fontsize = 24)\n",
+    "plt.ylabel(\"$\\Delta$y arcseconds\", fontsize = 18)\n",
+    "plt.xlabel(\"$\\Delta$x arcseconds\", fontsize = 18)\n",
+    "plt.colorbar(h[3], ax=ax)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "If I have a potential asteroid, it will have some true movement between the images. We would like a '5 sigma' detection of movement. What is that distance in arcseconds?\n",
+    "\n",
+    "What is the question in words?\n",
+    "What is the question in math? [Hint: Come up with an analytic description of your pdf(). The Blue Book may be helpful here.]\n",
+    "What distance in arcseconds represents a 5 sigma detection of motion?"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Statistical question: how much does the asteroid have to move to have a distance with a significant enough probability resulting in a '5 sigma' dtection of movement?"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "\n",
+    "Question 2, Part 1\n",
+    "Statistical Question: How far must a potential asteroid travel, measured in arcseconds, in order to produce a probability that is associated with 5 sigma?\n",
+    "\n",
+    "Question 2, Part 2\n",
+    "We know the probability of getting a detection of 5 sigma is about 1 in 3.5 million. So the integral we are interested in taking is that of the distribution above to find the lower bound such that the integral is equal to the 5 sigma probability.\n",
+    "$$\\begin{equation*}\n",
+    "\\int_{x}^{\\infty} pdf(x) \\,dx\n",
+    "\\end{equation*}$$\n",
+    "Where we are looking for x."
```

Comment:
Be sure to include why we are able to analyze this as a Rayleigh distribution, rather than a 2D Gaussian distribution